src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -699,6 +698,16 @@ private SpawnResult handleError(
         .build();
   }
 
+  static List<Property> getExtraPlatformProperties(Spawn spawn, Path execRoot) {
+    if (Spawns.shouldDifferentiateWorkspaceCache(spawn)) {
+      return ImmutableList
+          .of(Property.newBuilder().setName("bazel-workspace").setValue(execRoot.getBaseName())
```

Comment:
nit: make `"bazel-workspace"` also allude to the fact that it's only for splitting cache keys

Reply:
Changed to `"bazel-differentiate-workspace-cache"`.